release-a-chart-and-bump-to-upstream.md

```diff
@@ -0,0 +1,329 @@
+# Release a New Chart and Bump it to Upstream
+
+## Release a New Chart on Harvester Chart Repo
+
+This document demonstrates how to release a new harvester-cloud-provider chart version, and bump the new version to upstream.
+
+### Change an Existing Chart
+
+e.g. https://github.com/harvester/charts/pull/229
+
+merge to `master` branch
```

Comment:
I thought we merged `master` because the chart content changed? If so, could we provide a more specific description here?